### PR TITLE
COVID-19 TR Data branch merging.

### PR DESCRIPTION
**Pull request by egirgin**
I've implemented an endpoint and a unit test. Please review my code.

### CODE REVIEW

File: practice-app/backend/controllers/covid19.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+const Response = require('../utils/response');
+const Request = require('../utils/request');
+const Constants = require('../constants');
```
[the remark this answers is not in the document]
Comment (egirgin) [reply practice-app/backend/controllers/covid19.js]: I updated the endpoint and removed the require(Constansts).

**Comment by egirgin** [comment]
Guys, I updated my code so that it works well with ".env" file. Please check that out.

File: practice-app/backend/controllers/covid19.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                    TotalRecovered : dataTR.TotalRecovered,
+                    ActiveCases : dataTR.TotalConfirmed - (dataTR.TotalRecovered + dataTR.TotalDeaths),
+                    lastUpdate : dataTR.Date.substring(0,10)
+                } : {};
+
+                // Respond to the front-end.
+                Response.send(response, responseBody);
+
+            }
+            catch{ // If the maximum number of request reached, return message.
```
Comment (koraycetiin) [inline practice-app/backend/controllers/covid19.js:65]: this line causes error on my pc. it should be catch (varName) { i think

File: practice-app/backend/tests/covid19.spec.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+const chai = require('chai'),
+    mocha = require('mocha'),
+    app = require('../app').App,
+    Constants = require('../constants'),
```
Comment (berkayalkan) [inline practice-app/backend/tests/covid19.spec.js:4]: This line gives error when I run the npm test command. I think you forgot to delete constants part from this file

File: practice-app/backend/controllers/covid19.js (hunk 2 above)
Comment (egirgin) [reply practice-app/backend/controllers/covid19.js:65]: It worked without it in my environment but I added `catch(error){`.

File: practice-app/backend/tests/covid19.spec.js (hunk 3 above)
Comment (egirgin) [reply practice-app/backend/tests/covid19.spec.js:4]: You're right. I'm going to remove it.

**Pull request merged**